### DISCUSSION TITLE
OBJEX Link for sale!

### DISCUSSION THREAD

**Discussion by salvatoreraccardi** [salvatoreraccardi/OBJEX_LINK#8] [Announcements]
[BUY NOW!](https://www.tindie.com/products/objex/objex-link-rev15/)

**Comment by voxpelli** [comment]
Already sold out?

**Reply by salvatoreraccardi** [reply]
yes, in less than 1 hour. Maybe I have an extra unit I'll let you know. Either way, I'll be doing a restock next week.

**Reply by voxpelli** [reply]
Congrats on successful first sale! 🎉

**Reply by salvatoreraccardi** [reply]
Hi, if you want you can buy one kit: https://www.tindie.com/products/objex/objex-link-rev15/
